beta-src/src/components/map/WDMap.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from "react";
+import WDBoundaries from "./variants/classic/components/WDBoundaries";
+import WDCenters from "./variants/classic/components/WDCenters";
+import WDGameBoardOutlines from "./variants/classic/components/WDGameBoardOutlines";
+import WDNeutral from "./variants/classic/components/WDNeutral";
+import WDSeaAreas from "./variants/classic/components/WDSeaAreas";
+
+interface WDMapProps {
+  height: number;
+  preserveAspectRatio: string;
+  svgElement: any;
+  width: number;
+}
+
+const WDMap: React.FC<WDMapProps> = function ({
+  height,
+  preserveAspectRatio,
+  svgElement,
+  width,
+}): React.ReactElement {
+  return (
+    <svg
+      fill="none"
+      height={height}
+      preserveAspectRatio={preserveAspectRatio}
+      ref={svgElement}
+      width={width}
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <g id="full-map-svg">
+        <rect width={width} height={height} fill="white" />
+        <g id="container">
+          <g id="bg">
+            <path
+              id="Vector"
+              d="M4017.07 0H0V1991.02H4017.07V0Z"
+              fill="#FFF8F3"
+            />
+          </g>
+          <WDCenters />
+          <WDGameBoardOutlines />
+          <WDNeutral />
+          <WDSeaAreas />
+          <WDBoundaries />
+        </g>
+      </g>
+      <defs>
```

Comment:
should we break defs out to a separate import as well? my thinking is they might get bigger?